testcontainers/docker/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM hyperledger/iroha:1.1.0
+#ATTENTION PLEASE
+#this is my image and it should be changed to official before merging!!!!
```

Comment:
I suggest to add information why it is not official

Reply:
It is in PR description